Sources/Kitura/RouterElement.swift

```diff
@@ -124,6 +135,44 @@ class RouterElement {
             self.processHelper(request: request, response: response, next: next)
         }
     }
+    
+    /// Perform a simple match
+    ///
+    /// - Parameter path: the path being matched
+    /// - Parameter request: the request
+    /// - Parameter response: the router response
+    /// - Parameter next: the closure for the next execution block
+    private func performSimpleMatch(path: String, request: RouterRequest, response: RouterResponse, next: @escaping () -> Void) {
+        let pathToMatch = path.isEmpty ? "/" : path
+        var matched: Bool
+        let matchedPath: String
+        
+        if allowPartialMatch {
+            matched = pathToMatch.hasPrefix(pattern!)
+            if matched && pattern! != "/" {
```

Comment:
I would add a `guard let pattern = pattern else` in the beginning of the function and would not use `pattern!`